vmdb/app/models/file_depot.rb

```diff
@@ -25,6 +25,7 @@ def requires_support_case?
   end
 
   def depot_hash=(hsh = {})
+    ActiveSupport::Deprecation.warn "FileDepot#depot_hash= is depricated, please create an instance of the proper subclass"
```

Comment:
typo: `deprecated`...same below.

For deprecations, we've tried to send to $log as well as Kernel.warn.  I don't know where ActiveSupport::Deprecation sends to.